bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/DefaultSystemChannelTypeProvider.java

```diff
@@ -397,14 +398,24 @@ public class DefaultSystemChannelTypeProvider implements ChannelTypeProvider {
                     StateDescriptionFragmentBuilder.create().withReadOnly(true).withPattern("%.1f %unit%").build())
             .withTags(List.of("Measurement", "Energy")).build();
 
+    /**
+     * UV Index: system wide {@link ChannelType} which shows the UV Index
+     */
+    public static final ChannelType SYSTEM_UV_INDEX = ChannelTypeBuilder
+            .state(SYSTEM_CHANNEL_TYPE_UID_UV_INDEX, "UV Index", CoreItemFactory.NUMBER)
+            .withDescription("Current UV Index")
+            .withStateDescriptionFragment(StateDescriptionFragmentBuilder.create().withReadOnly(true)
+                    .withPattern("%.0f/16").withMinimum(BigDecimal.ZERO).withMaximum(new BigDecimal(16)).build())
```

Comment:
I just read the minimum is 1 and the maximum is 11+. Are you sure about 16 ?

Reply:
I believe there is no real max.

Reply:
Correct. It's the same as the "Richter scale" for earthquakes.

Reply:
So you should not define a max and the pattern "x/16" should be changed too.